tests/documentation/DocumentationStyleBearTest.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from bears.documentation.DocumentationStyleBear import DocumentationStyleBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+
+def load_testfiles(test, good_file, bad_file):
+    good_file_path = os.path.join(os.path.dirname(__file__), "test_files",
+                                  test, good_file)
+    bad_file_path = os.path.join(os.path.dirname(__file__), "test_files",
+                                 test, bad_file)
+    with open(bad_file_path) as bad_f, open(good_file_path) as good_f:
+        return good_f.read(), bad_f.read()
```

Comment:
haha yeah this is way better.

but I'd also like to have a "test" parameter, because each of the docmentation bears will have tests like this, and I will also be moving this function to a testutils module in a future PR.